be/src/vec/exec/vorc_reader.h

```diff
@@ -0,0 +1,57 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+#pragma once
+
+#include <arrow/adapters/orc/adapter.h>
+#include <arrow/api.h>
+#include <arrow/buffer.h>
+
+#include <stdint.h>
+#include <map>
+#include <string>
+#include "common/status.h"
+#include "exec/arrow_reader.h"
+namespace doris::vectorized {
+
+// Reader of orc file
+class VORCReaderWrap : public ArrowReaderWrap {
```

Comment:
May be the file should in `be/exec` same as `parquet_reader`. And the class do not use vectorized::block. maybe no need name `VORC`?